python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -68,47 +61,49 @@ We denote by :math:`E_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` the
-elliptical distribution
-:math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}`.
+-  An *elliptical copula* :math:`C^E_{\mat{R},\psi}` is the copula of an
+   elliptical distribution :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}`.
 
-The *generic elliptical representative* of an elliptical distribution
-family :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` is the
-elliptical distribution whose cumulative distribution function is
-:math:`E_{\vect{0},\vect{1},\mat{R},\psi}`.
+-  The *generic elliptical representative* of an elliptical distribution
+   family :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` is the
+   elliptical distribution whose cumulative distribution function is
+   :math:`E_{\vect{0},\vect{1},\mat{R},\psi}`.
 
-The *standard spherical representative* of an elliptical distribution
-family :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` is the
-spherical distribution whose cumulative distribution function is
-:math:`E_{\vect{0},\vect{1},\mat{I}_n,\psi}`.
+-  The *standard spherical representative* of an elliptical distribution
+   family :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` is the
+   spherical distribution whose cumulative distribution function is
+   :math:`E_{\vect{0},\vect{1},\mat{I}_n,\psi}`.
 
-The family of distributions with marginal cumulative distribution
-functions are :math:`F_1,\dots,F_n` and any elliptical copula
-:math:`C^E_{\mat{R},\psi}` is denoted by
-:math:`{\cD}_{F_1,\dots,F_n,C^E_{\mat{R},\psi}}`. The cumulative
-distribution function of this distribution is noted
-:math:`D_{F_1,\dots,F_n,C^E_{\mat{R},\psi}}`.
+-  The family of distributions with marginal cumulative distribution
+   functions are :math:`F_1,\dots,F_n` and any elliptical copula
+   :math:`C^E_{\mat{R},\psi}` is denoted by
+   :math:`{\cD}_{F_1,\dots,F_n,C^E_{\mat{R},\psi}}`. The cumulative
+   distribution function of this distribution is noted
+   :math:`D_{F_1,\dots,F_n,C^E_{\mat{R},\psi}}`.
 
 The random vector :math:`\vect{X}` is supposed to be continuous and
 with full rank. It is also supposed that its cumulative marginal
 distribution functions :math:`F_i` are strictly increasing (so they
 are bijective) and that the matrix :math:`\mat{R}` of its elliptical
 copula is symmetric positive definite.
 
-**Generalized Nataf transformation**: Let :math:`\vect{X}` in
+Generalized Nataf transformation:
+---------------------------------
+
+Let :math:`\vect{X}` in
 :math:`\Rset^n` be a continuous random vector following the
 distribution :math:`D_{F_1,\dots,F_n,C^E_{\mat{R},\psi}}`. The
 *Generalized Nataf transformation* :math:`T_{Nataf}^{gen}` is defined
 by:
 
-.. math:: \vect{u} = T_{Nataf}^{gen}(\vect{X})=T_3\circ T_2\circ T_1(\vect{X})
+.. math::
+
+    \vect{u} = T_{Nataf}^{gen}(\vect{X})=T_3\circ T_2\circ T_1(\vect{X})
 
-where the three transformations :math:`T_1`, :math:`T_2` and
-:math:`T_3` are given by:
+where the three transformations :math:`T_1`, :math:`T_2` and :math:`T_3` are given by:
```

Comment:
The equations do not format as expected:
![image](https://user-images.githubusercontent.com/31351465/233780216-84c3ba22-0a32-4948-b907-fce1624e29c5.png)

Reply:
I modified the format.